File: evals/refs/vowels.py
import sys

out = []
for line in sys.stdin:
    line = line.rstrip('\n')
    count = sum(1 for c in line if c.lower() in 'aeiou')
    out.append(str(count))
sys.stdout.write('\n'.join(out) + ('\n' if out else ''))

File: evals/refs/vowels.rs
use std::io::{self, BufRead, Write};

fn main() {
    let stdin = io::stdin();
    let stdout = io::stdout();
    let mut out = stdout.lock();
    for line in stdin.lock().lines() {
        let line = line.unwrap();
        let count = line
            .chars()
            .filter(|c| matches!(c.to_ascii_lowercase(), 'a' | 'e' | 'i' | 'o' | 'u'))
            .count();
        writeln!(out, "{}", count).unwrap();
    }
}
